tests/conftest.py

```diff
@@ -1,43 +1,73 @@
+from dataclasses import dataclass
 import logging
 import pytest
 import shutil
 from multiprocessing import Process
 from pathlib import Path
-from typing import Tuple
 
-from tests.utils import run_http_server
+from tests.utils import SlotMeta, run_http_server
 
 logger = logging.getLogger(__name__)
 
-# not enable proxy when doing test
-DEFUALT_PROXY_INFO = """
-enable_local_ota_proxy: false
-"""
 
-OTA_IMAGE_DIR = "/ota-image"
-OTA_IMAGE_SERVER_ADDR = "127.0.0.1"
-OTA_IMAGE_SERVER_PORT = 8080
+@dataclass
+class TestConfiguration:
+    # dummy ota-image setting
+    OTA_IMAGE_DIR = "/ota-image"
+    OTA_IMAGE_SERVER_ADDR = "127.0.0.1"
+    OTA_IMAGE_SERVER_PORT = 8080
+    KERNEL_VERSION = "5.8.0-53-generic"
+    CURRENT_VERSION = "123.x"
+    UPDATE_VERSION = "789.x"
 
+    # slots settings for testing
+    SLOT_A_UUID = "aaaaaaaa-0000-0000-0000-aaaaaaaaaaaa"
+    SLOT_A_PARTUUID = SLOT_A_UUID
```

Comment:
In general, UUID and PARTUUID are different values.

Reply:
Yes, use the same value is only for test convenience, `grub` use `UUID` and `cboot` use `PARTUUID`. So I use different var name here.

Reply:
Could you add the comment, why UUID and PARTUUID can be the same value?

Reply:
Sure!